Apps/ValidationTests/Scripts/validation_native.js

```diff
@@ -38,6 +38,8 @@ function compare(test, renderData, referenceImage, threshold, errorRatio) {
 
     if (differencesCount) {
         console.log("Pixel difference: " + differencesCount + " pixels.");
+    } else {
+        console.log("No pixel difference !!!");
```

Comment:
Nit: Remove space between difference and !!!

Reply:
Done